lambdas/shared/t4_lambda_shared/decorator.py

```diff
@@ -4,12 +4,17 @@
 
 from base64 import b64decode, b64encode
 from functools import wraps
+import gzip
 import json
 import traceback
 
 from jsonschema import Draft4Validator, ValidationError
 
 
+GZIP_MIN_LENGTH = 1024
```

Comment:
We need to keep this in sync with the corresponding API Gateway, right? i.e. worth a comment

Reply:
Nah, that's just to avoid gzip'ing tiny responses, where the compressed data will probably be bigger than the original